### PR TITLE
Add several archs to labeler

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Adds MSP430 to the labeler.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Review by wargio — APPROVED** [review]

**Review by XVilka — CHANGES_REQUESTED** [review]
In fact, not only MSP430 was missing, but many architectures were missing. It makes sense to add them all at once.

**Comment by Rot127** [comment]
Labels are added now.

File: .github/labeler.yml [hunk 1]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/dcpu16/**/*
+  - librz/arch/p/asm/_*dcpu16*_
+  - librz/arch/p/analysis/_*dcpu16*_
+
+EBC:
+  - librz/arch/is*/ebc/**/*
+  - librz/arch/p/asm/_*ebc*_
+  - librz/arch/p/analysis/_*ebc*_
+
+GB:
```
Comment (XVilka) [inline .github/labeler.yml]: Maybe better GameBoy?

File: .github/labeler.yml [hunk 2]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/gb/**/*
+  - librz/arch/p/asm/_*gb*_
+  - librz/arch/p/analysis/_*gb*_
+
+h8300:
```
Comment (XVilka) [inline .github/labeler.yml]: H8300

File: .github/labeler.yml [hunk 3]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/lh5801/**/*
+  - librz/arch/p/asm/_*lh5801*_
+  - librz/arch/p/analysis/_*lh5801*_
+
+LM32:
+  - librz/arch/is*/lm32/**/*
+  - librz/arch/p/asm/_*lm32*_
+  - librz/arch/p/analysis/_*lm32*_
+
+LUAC:
```
Comment (XVilka) [inline .github/labeler.yml]: Just Lua is probably better, also shorter

File: .github/labeler.yml [hunk 4]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/tms320/**/*
+  - librz/arch/p/asm/_*tms320*_
+  - librz/arch/p/analysis/_*tms320*_
+
+TriCore:
+  - librz/arch/is*/tricore/**/*
+  - librz/arch/p/asm/_*tricore*_
+  - librz/arch/p/analysis/_*tricore*_
+
+v810:
```
Comment (XVilka) [inline .github/labeler.yml]: V810

File: .github/labeler.yml [hunk 5]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/v810/**/*
+  - librz/arch/p/asm/_*v810*_
+  - librz/arch/p/analysis/_*v810*_
+
+v850:
```
Comment (XVilka) [inline .github/labeler.yml]: V850

File: .github/labeler.yml [hunk 6]
```diff
@@ -218,6 +228,226 @@ PPC:
...
+  - librz/arch/is*/6502/**/*
+  - librz/arch/p/asm/_*6502*_
+  - librz/arch/p/analysis/_*6502*_
+
+8051:
+  - librz/arch/is*/8051/**/*
+  - librz/arch/p/asm/_*8051*_
+  - librz/arch/p/analysis/_*8051*_
+
+AMDAm29000:
```
Comment (XVilka) [inline .github/labeler.yml]: Just Am29k

**Review by XVilka — CHANGES_REQUESTED** [review]
LGTM with a few small changes

**Review by XVilka — APPROVED** [review]

**Pull request merged**